pkg/controller/machineautoscaler/machinetarget_test.go

```diff
@@ -283,3 +283,70 @@ func TestNamespacedName(t *testing.T) {
 			nn.Namespace, TargetNamespace)
 	}
 }
+
+func TestHasGPUCapacity(t *testing.T) {
+	target := NewTarget()
+	target.SetAnnotations(map[string]string{
+		autoscalerCapacityGPU: "1",
+	})
+
+	if !target.HasGPUCapacity() {
```

Comment:
Should each of these cases be a subtest?

Reply:
yeah, agreed